package.json

```diff
@@ -29,7 +31,8 @@
   "scripts": {
     "start": "NODE_ENV=development ./node_modules/.bin/webpack-dev-server --watch",
     "build": "NODE_ENV=production ./node_modules/.bin/webpack --config ./webpack.config.js --display-max-modules 0",
-    "prepublish": "npm run build"
+    "prepublish": "npm run build",
+    "dev": "cross-env NODE_ENV=development ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config ./test/webpack.config.js --progress --colors --content-base test/"
```

Comment:
I've added a started kit (covers local dev server and webpack config). So this will not be required 😄